community/community-contacts.md

```diff
@@ -10,4 +13,11 @@ The list is open to anyone for subscription, and everyone is welcome to join MEI
 
 Also, the archives of the list are open to the public and can be browsed online: [https://lists.uni-paderborn.de/pipermail/mei-l/](https://lists.uni-paderborn.de/pipermail/mei-l/).
 
-Normally, there is not much traffic on the list, but the tone is friendly and most questions are answered within the first 24 hours. So if you have a particular question about MEI, please let others learn from your responses as well by asking there in public.
\ No newline at end of file
+# Slack channel
+
+MEI has a Slack team that is used for discussion, questions, announcements, and interest groups.
```

Comment:
There isn't anywhere mentioned the name of or a link to the  actual Slack channel. Is this by purpose?  (Would be good for users who do not use it very frequently.)